// Copyright 2021 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package side_search;

// Used for storing a tab's associated Side Search state.
message SideSearchTabData {
  // The last search URL encountered by the current tab.
  string last_search_url = 1;

  // Whether the side panel is open or closed for the current tab.
  bool toggled_open = 2;
}
